src/brogue/Globals.h

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
Globals.h contains global (mostly-constants) that might be reasonably desired to be overriden in variants, e.g. player or interface colors and the tables that define the game objects. Variables should mostly be const, but in many cases the large tables of consts are also used to store a small amount of mutable state (so the tables cannot be const). A better pattern for the future is to break these tables into separate const and mutable tables.

Reply:
How could they be overridden if they should be const?

Personally I think we should take out any variable that isn't actually changed, and put them in GlobalsBase. No point guessing an API we're not using yet. It's easy to move them around as-needed, right?

Reply:
We could collapse GlobalsBase and Globals back together and just group them separately. There is a logical separation between the variables in both files, but I wouldn't say I've necessarily got the final assignment here.

Reply:
My comment re: const; I think the kind of tables that define a load of constants around how objects in the game work would ideally be like `charmEffectTableEntry` - all const members and address through a `const` pointer, e.g. `const charmEffectTableEntry *charmEffectTable;`. However, this is not possible for most of the tables since, as well as storing data about objects, they also are used to set some flags during the game. For example, `dungeonFeatureCatalog` is mostly full of constants but has `messageDisplayed` which is updated during the game. I'd separate this mutable flag into a seperate array of structs, `dungeonFeatureState` or similar.

Reply:
I'm not fussed about one vs two files, just think we shouldn't preemptively add any values as variant-specific that aren't actually changed in any variant yet, is all

Reply:
Is that true? I thought all these colors at the top of the file weren't different between CE and RB? I guess it doesn't matter though, not a blocker